docs/general/server/media/shows.md

```diff
@@ -68,6 +68,16 @@ Episode numbering for specials may vary from metadata provider to metadata provi
 
 :::
 
+Specials can also be shown within a Season if so desired. This can be helpful when they are part of a continued storyline during the Season. This requires 2 settings:
+
+1. The option under Dashboard -> Library -> Display -> "Display specials within their series they aired in" must be enabled
+2. The Season and Episode they aired before/after must be set within the metadata.
+    - These can be set in the Metadata editor or in an NFO using the `airsbefore_season`, `airsafter_season`, and `airsbefore_episode` tags.
+    - The `Airs before series` field can be set to show (and play) the Special at the start of the Season. It can also be combined with the `Airs before episode` field to insert it as a Mid-Season Special before the specified episode.
```

Comment:
```suggestion
    - The `Airs before series` field can be set to show (and play) the special at the start of the season. It can also be combined with the `Airs before episode` field to insert it as a mid-season special before the specified episode.
```